### PR TITLE
Add serveraddr/addr.go unit test

### PR DESCRIPTION
**Pull request by BearLu**

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@BearLu Thank you for uploading pr.  how about rebase these three commits into one commit?

**Comment by rambohe-ch** [comment]
@BearLu please sign CLA

File: pkg/yurttunnel/server/serveraddr/addr_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+	ns :="hello"
+	name := "world"
+	domains = getDefaultDomainsForSvc(ns, name)
+	if len(domains) == 0 {
+		t.Log("domains len is zero")
+	} else {
+		if len(domains) == 4{
+			// expected, actual
+			assert.Equal(t, name, domains[0])
```
Comment (rambohe-ch) [inline pkg/yurttunnel/server/serveraddr/addr_test.go:23]: maybe it does not good to import new lib(`github.com/stretchr/testify/assert`), how about only use `testing` to replace it?

File: pkg/yurttunnel/server/serveraddr/addr_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package serveraddr
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+)
+
+func TestGetDefaultDomainsForSvc(t *testing.T) {
+	domains := getDefaultDomainsForSvc("", "")
```
Comment (rambohe-ch) [inline pkg/yurttunnel/server/serveraddr/addr_test.go:10]: you can add a new test case or a sub test case for ns and name empty setting.

File: pkg/yurttunnel/server/serveraddr/addr_test.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import (
+	"fmt"
+	"testing"
+)
+
+func TestGetDefaultDomainsForSvcInputParamEmptyChar(t *testing.T) {
+	domains := getDefaultDomainsForSvc("", "")
+	if len(domains) == 0 {
+		t.Log("domains len is zero")
+	}
```
Comment (rambohe-ch) [inline pkg/yurttunnel/server/serveraddr/addr_test.go:27]: if length of domains is not 0, the case is also passed. so it is better to use `if len(domains) != 0` verification and then `t.Errorf()` to output error message.

Comment (BearLu) [reply pkg/yurttunnel/server/serveraddr/addr_test.go:27]: OK,I will modify it later.

File: pkg/yurttunnel/server/serveraddr/addr_test.go (hunk 1 above)
Comment (BearLu) [reply pkg/yurttunnel/server/serveraddr/addr_test.go]: I use testing replace it

File: pkg/yurttunnel/server/serveraddr/addr_test.go (hunk 2 above)
Comment (BearLu) [reply pkg/yurttunnel/server/serveraddr/addr_test.go:24]: I modified it

File: pkg/yurttunnel/server/serveraddr/addr_test.go (hunk 3 above)
Comment (BearLu) [reply pkg/yurttunnel/server/serveraddr/addr_test.go:27]: I modified it

**Comment by rambohe-ch** [comment]
LGTM
